src/components/Restoration/YourCompanySummary.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <v-card flat id="restoration-summary">
+    <!-- Section Header -->
+    <div class="summary-header px-4 mb-2 rounded-t">
+      <v-row no-gutters>
+        <v-col cols="9">
+          <img class="header-icon" src="@/assets/images/currency-usd-circle.svg">
+          <label class="summary-title">Your Company</label>
+        </v-col>
+      </v-row>
+    </div>
+    <!-- Business Name -->
+    <template>
+      <v-divider class="mx-4" />
+      <div class="section-container business-name-summary">
+        <v-row no-gutters>
+          <v-col cols="3">
+            <label><strong>Restoration Type</strong></label>
+          </v-col>
+          <v-col cols="8" class="mt-n1" v-if="isLimitedExtendRestorationFiling">
+            <div class="font-weight-bold">Limited Restoration Extension</div>
+            <div>Extension Time: {{ getFormattedExpiryText }}</div>
+          </v-col>
+          <v-col cols="8" class="mt-n1" v-if="isLimitedConversionRestorationFiling">
+            <div class="font-weight-bold">Conversion to Full Restoration</div>
+            <div>[TODO - Applicant's relationship: Director, Shareholder]</div>
+          </v-col>
+        </v-row>
+        <v-row no-gutters class="mt-3" v-if="hasFileNumber">
+          <v-col cols="3">
+            <label><strong>Approval Type</strong></label>
+          </v-col>
+          <v-col cols="8" class="mt-n1">
+            <div class="font-weight-bold">Approved by Court Order</div>
+            <div>Court Order Number: {{ getFileNumber }}</div>
+          </v-col>
+        </v-row>
+      </div>
+    </template>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { mapGetters } from 'vuex'
+
+export default {
+  computed: {
+    ...mapGetters(['isLimitedConversionRestorationFiling', 'isLimitedExtendRestorationFiling',
+      'hasFileNumber', 'getFileNumber', 'getFormattedExpiryText'])
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.summary-header {
+  display: flex;
+  background-color: $BCgovBlue5O;
+  padding: 1.25rem;
+}
+
+.summary-title {
+  padding-left: 0.5rem;
```

Comment:
After much UI cleanup in the past, I favour using the [Vuetify spacing classes](https://vuetifyjs.com/en/styles/spacing/) right in the template for whitespace and general layout. This is way easier to understand and tweak than needing to look up the CSS for the `summary-title` and all other classes. Thoughts?

Reply:
Yes, I agree, I prefer that too.  I'll change it.